src/main/java/dev/openfeature/javasdk/internal/StructureFieldType.java

```diff
@@ -0,0 +1,5 @@
+package dev.openfeature.javasdk.internal;
+
+public enum StructureFieldType {
```

Comment:
This isn't used now that there pairs were removed?

Reply:
Yep, great catch.

Reply:
Removed.